tests/test_migrators.py

```diff
@@ -746,6 +746,169 @@
     - ocefpaf
 """
 
+sample_r_base = """
+{% set version = '0.7-1' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-stabledist
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: stabledist_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/stabledist_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/stabledist/stabledist_{{ version }}.tar.gz
+
+
+  sha256: 06c5704d3a3c179fa389675c537c39a006867bc6e4f23dd7e406476ed2c88a69
+
+build:
+  number: 1
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  skip: True  # [win32]
+
+requirements:
+  build:
+    - r-base
+
+  run:
+    - r-base
+
+test:
+  commands:
+    - $R -e "library('stabledist')"  # [not win]
+    - "\"%R%\" -e \"library('stabledist')\""  # [win]
+"""
+
+updated_r_base = """
+{% set version = '0.7-1' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-stabledist
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: stabledist_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/stabledist_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/stabledist/stabledist_{{ version }}.tar.gz
+
+
+  sha256: 06c5704d3a3c179fa389675c537c39a006867bc6e4f23dd7e406476ed2c88a69
+
+build:
+  noarch: generic
+  number: 2
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  skip: True  # [win32]
```

Comment:
@isuruf I thought we were getting rid of the line about skipping the win32 build. Should that be reflected in these tests or no?

Reply:
Doesn't matter if it were there or not. conda-build will ignore this. I suggested removing it because, people might get confused. If it's already there, keeping it there won't matter.

Reply:
@isuruf Got it. Thanks!

Reply:
I guess the linter will e a bit unhappy about this but I guess we can remove it.